bindings_generator/src/main.rs

```diff
@@ -42,4 +43,29 @@ impl ParseCallbacks for RenameCallback {
         item.strip_suffix("__bindgen_ty_1")
             .map(|name| format!("{name}_u"))
     }
+
+    fn enum_variant_name(
+        &self,
+        enum_name: Option<&str>,
+        original_variant_name: &str,
+        _variant_value: EnumVariantValue,
+    ) -> Option<String> {
+        // Remove the enum name from the variant name:
+        // `IRShaderStage::IRShaderStageVertex` -> `IRShaderStage::Vertex`
+        if let Some(enum_name) = enum_name {
+            let enum_name = enum_name
+                .strip_prefix("enum ")
+                .unwrap()
+                .replace("Flags", "Flag");
+            let new_name = original_variant_name
+                .strip_prefix(&enum_name)
+                .unwrap_or(original_variant_name);
+            let mut new_name = new_name.strip_prefix("_").unwrap_or(new_name).to_string();
```

Comment:
What's this `strip_prefix` for?

Reply:
Assuming you mean the one on line 63: some enum variants for some reason have an underscore in there `IROperatingSystem_macOS`, which would result in `IROperatingSystem::_macOS` instead of `IROperatingSystem::macOS`

Reply:
Cool so it's not automatically inserted already for number prefixes.